less/components/signup-form.less

```diff
@@ -27,6 +27,7 @@
   input, select {
     font-family: Fira Sans Condensed;
     letter-spacing: 0.03rem;
+    font-size: 90%;
```

Comment:
Can you try `font-size: 0.9rem;` and see if that roughly gets the same result? We're using rems for fonts in general in this site. The advantage is when the screen size gets smaller, so does the font.